### PR TITLE
[Newton] DOCS-4 Add release notes for Ironic.

### PR DESCRIPTION
**Pull request by cjloader**
Issue: [DOCS-4](https://rpc-openstack.atlassian.net/browse/DOCS-4)

### CODE REVIEW

File: releasenotes/notes/added-support-for-ironic-05361d144fce5dcf.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
```
Comment (prometheanfire) [inline releasenotes/notes/added-support-for-ironic-05361d144fce5dcf.yaml]: probably don't need the `|` if it's just one line, but it's just a minor nit.

**Comment by cjloader** [comment]
"Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?"

recheck_cit_xenial_swift_deploy
recheck_cit_trusty_swift_deploy

**Comment by cjloader** [comment]
recheck_cit_xenial_swift_deploy
recheck_cit_trusty_swift_deploy

**Comment by cjloader** [comment]
recheck_cit_xenial_swift_deploy
recheck_cit_trusty_swift_deploy

**Comment by cjloader** [comment]
recheck_cit_xenial_swift_deploy
recheck_cit_trusty_swift_deploy

**Pull request merged**